packages/datastore/src/client.ts

```diff
@@ -172,11 +158,9 @@ export class CollaborationClient extends WSConnection<
     if (token) {
       queryParams.push(`token=${encodeURIComponent(token)}`);
     }
-    if (this._storeId !== null) {
-      queryParams.push(
-        `storeId=${encodeURIComponent(this._storeId.toString(10))}`
-      );
-    }
+    // queryParams.push(
```

Comment:
Is this intended?

Reply:
I heard you don't like dead code :) I'll delete this.